app/templates/buyers/dashboard.html

```diff
@@ -54,7 +54,7 @@
 
 {{ summary.heading("Unpublished requirements", id="unpublished_requirements") }}
 {% call(item) summary.list_table(
-    draft_briefs,
+    draft_briefs|sort(attribute='createdAt', reverse=True),
```

Comment:
I would have a slight preference for doing the sort in the backend rather than the template. I doubt we'd gain much speed-wise but as we're splitting out the briefs into separate lists in the backend, does it make sense to keep the ordering logic there as well?